src/main/scala/sbtassembly/AssemblyPlugin.scala

```diff
@@ -151,4 +155,5 @@ case class AssemblyOption(assemblyDirectory: File,
   prependShellScript: Option[Seq[String]] = None,
   maxHashLength: Option[Int] = None,
   shadeRules: Seq[ShadeRule] = Seq(),
-  level: Level.Value)
+  level: Level.Value,
+  isScala213AndLater: Boolean = false)
```

Comment:
Instead of passing a flag here, it might be better to pass Scala version directly in case we have to do more later, like "2.14 and later", "3.0 and later", etc.

Reply:
Also it's a personal preference, but could you keep `level` in the last position, and add scalaVersion above it?